### PR TITLE
Added a note about the `--overwrite` flag in getting started guide

### PR DESCRIPTION
**Pull request by kikisaints**
Added a note about the `--overwrite` flag.

### CODE REVIEW

**Comment by harinikmsft** [comment]
@acoates-ms  - can you confirm the verbiage looks right?

File: docs/getting-started.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ Lastly, install the React Native for Windows packages.
 npx react-native-windows-init --overwrite
 ```
 
+> The --overwrite flag is a temporary measure that ensures the correct files are copied to metro.config.js for the metro bundler to work with Windows. If you are starting a new app, this should have no impact. If you are adding Windows to your existing app and you have modified the metro.config.js file, please back up your changes, run the command and copy over to take effect.
+
 #### Confirming Metro JS reinstallation
```
Comment (harinikmsft) [inline docs/getting-started.md:43]: What is this section header pointing to? I dont see any content under it?

Comment (kikisaints) [reply docs/getting-started.md:43]: it's not a section header, ">" are inline notes.

Comment (harinikmsft) [reply docs/getting-started.md:43]: Sorry i was referring to "confirming Metro JS reinstallation"

File: docs/getting-started.md [hunk 2]
```diff
@@ -38,6 +38,8 @@ Lastly, install the React Native for Windows packages.
 npx react-native-windows-init --overwrite
 ```
 
+> The --overwrite flag is a temporary measure that ensures the correct files are copied to metro.config.js for the metro bundler to work with Windows. If you are starting a new app, this should have no impact. If you are adding Windows to your existing app and you have modified the metro.config.js file, please back up your changes, run the command and copy over to take effect.
```
Comment (chrisglein) [inline docs/getting-started.md:41]: "temporary issue" should link to an issue tracking this being different.

Comment (kikisaints) [reply docs/getting-started.md:41]: do we have an issue on the main repo tracking that or should I go ahead and create one?

File: docs/getting-started.md (hunk 1 above)
Comment (kikisaints) [reply docs/getting-started.md:43]: oh whoops! this was an artifact left over from an edit on this pr. Thanks for catching, I'll remove.

File: docs/getting-started.md (hunk 2 above)
Comment (chrisglein) [reply docs/getting-started.md:41]: I don't think we have one. Go ahead and create one and link to it here.

Comment (jonthysell) [reply docs/getting-started.md:41]: Now that the site is versioned, you should add this text to the getting-started guides under 60 and 61.

Comment (kikisaints) [reply docs/getting-started.md:41]: created it here: https://github.com/microsoft/react-native-windows/issues/4698

**Pull request merged**